lib/profile/profilesRouter.js

```diff
@@ -56,6 +56,15 @@ const cmdDefinitions = [
         .positional('name', { describe: 'Profile name' });
     },
     handlerName: 'deleteProfile'
+  },
+  {
+    command: 'login [name]',
+    desc: 'Acquire new authentication token',
```

Comment:
I think "tokens" is not something that the user knows or cares about. It should be something like:
"Re-authenticate a profile. If no profile is specified, the active profile is used."
Similarly, we need to change the description of the "show" command:
"View detailed info for a profile. If no profile is specified, the active profile is used.